src/elements/common/sub-header/Sort.js

```diff
@@ -42,15 +42,14 @@ const Sort = ({ sortBy, sortDirection, onSortChange }: Props) => (
                 const sortItemKey = `${sortByValue}${sortDirectionValue}`;
 
                 return (
-                    <MenuItem
+                    <SelectMenuItem
```

Comment:
use SelectMenuItem because it's literally MenuItem with `isSelectItem` 
https://github.com/box/box-ui-elements/blob/fdb695403768eed1f2d592e04bc802f1ef755b2d/src/components/menu/SelectMenuItem.tsx#L5